lib/bike_brigade/slack_api.ex

```diff
@@ -7,7 +7,13 @@ defmodule BikeBrigade.SlackApi do
 
   @callback post!(url, body, headers) :: :ok
 
-  @headers [{"content-type", "application/json"}]
+  @token BikeBrigade.Utils.fetch_env!(:slack, :token)
```

Comment:
this may not work in production -- the module attribute `@token` is set at compile time, and compilation happens in an environment without the secrets which are loaded at runtime.

I think you need to put the token in the headers at runtime